include/awkward/Content.h

```diff
@@ -1,77 +1,107 @@
-// BSD 3-Clause License; see https://github.com/jpivarski/awkward-1.0/blob/master/LICENSE
+// BSD 3-Clause License; see
+// https://github.com/jpivarski/awkward-1.0/blob/master/LICENSE
 
 #ifndef AWKWARD_CONTENT_H_
 #define AWKWARD_CONTENT_H_
 
 #include <cstdio>
 
-#include "awkward/cpu-kernels/util.h"
 #include "awkward/Identity.h"
 #include "awkward/Slice.h"
+#include "awkward/cpu-kernels/util.h"
 #include "awkward/io/json.h"
 #include "awkward/type/Type.h"
 
 namespace awkward {
-  class Content {
-  public:
-    Content(std::shared_ptr<Identity> id, std::shared_ptr<Type> type): id_(id), type_(type) { }
-    virtual ~Content() { }
+class Content {
+public:
+  Content(std::shared_ptr<Identity> id, std::shared_ptr<Type> type)
+      : id_(id), type_(type) {}
+  virtual ~Content() {}
```

Comment:
This indentation thing: let's turn it off. I'm using the indentation that comes with Atom, and Emacs seems to agree with it.

There's a style right now and it's consistent and motivated by the editors we use. (Do you have a favorite editor? Will it work with the indentation we have now?) This is an example of something I don't want to change.